projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts

```diff
@@ -24,4 +24,145 @@ describe('GoButtonComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('clicked()', () => {
+    it('sets isProcessing to false if useLoader is false', () => {
```

Comment:
Do we need this test now that we've got the one below it?

Reply:
Yep, I wanted to test things in both cases.